### PR TITLE
add: translate items in strings.xml

### PR DESCRIPTION
**Pull request by skyleaworlder**
截屏2022-07-24 上午11 36 03

截屏2022-07-24 上午11 36 25

Please check, and I'll update if any mistakes.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -51,17 +51,17 @@
...
-    <string name="login_subtitle">使用学校身份信息登录旦夕</string>
-    <string name="logging_in">登录中</string>
-    <string name="weak_password_note">服务器错误，认为您所输入的是弱密码。请重试数次，或修改密码后再登录。</string>
-    <string name="under_maintenance_note">网络错误，您尚未连接到复旦内网，无法执行登录。请开启校园 VPN 后重试。</string>
-    <string name="need_captcha_note">服务器错误，要求您输入验证码。请用手机浏览器打开 uis.fudan.edu.cn ，手动完成一次登录。</string>
-    <string name="invalid_credentials_note">ID 或密码不正确。请确认后再输入。</string>
-    <string name="id_non_empty">ID 不能为空</string>
-    <string name="password_non_empty">密码不能为空</string>
+    <string name="login_subtitle">Login with your UIS account</string>
+    <string name="logging_in">Logging</string>
```
Comment (w568w) [inline app/src/main/res/values/strings.xml:60]: `Logging in` will be better.

Comment (w568w) [inline app/src/main/res/values/strings.xml]: `Logging in` will be better.

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -51,17 +51,17 @@
...
+    <string name="weak_password_note">Server error: weak password entered. Please try again or change password and log in</string>
```
Comment (w568w) [inline app/src/main/res/values/strings.xml]: How about `Server error: weak password detected. Please retry or change your password`?

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -51,17 +51,17 @@
...
+    <string name="under_maintenance_note">Network error: not yet connected to Fudan intranet and failed to log in. Please open Fudan VPN and try again.</string>
```
Comment (w568w) [inline app/src/main/res/values/strings.xml]: 开启翻译成 `connect to` 更好。

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (skyleaworlder) [reply app/src/main/res/values/strings.xml]: Fix it!

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (skyleaworlder) [reply app/src/main/res/values/strings.xml]: Fixed!

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (skyleaworlder) [reply app/src/main/res/values/strings.xml]: OK, this is better.

**Review by w568w — APPROVED** [review]

**Pull request merged**